.vscode/settings.json

```diff
@@ -1,5 +1,9 @@
-{
-    "search.exclude": {
-        "src/**": true
-    }
-}
\ No newline at end of file
+{
+    "search.exclude": {
+        "src/**": true
+    },
+    "editor.codeActionsOnSave": {
+        "source.fixAll.markdownlint": true
```

Comment:
What happens if the markdownlint plugin is not installed?

Reply:
Just tested, and nothing happens. Like no error or anything. Also the extension is suggested by our environment so I'd go for it